apps/video/test/marionette/play_multiple_types_test.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var assert = require('assert'),
+    Video = require('./lib/video');
+
+marionette('play various types of videos', function() {
+
+  var app, client, selectors;
+
+  client = marionette.client({
+    profile: {
+      prefs: {
+        'device.storage.enabled': true,
+        'device.storage.testing': true,
+        'device.storage.prompt.testing': true
+      }
+    }
+  });
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+    selectors = Video.Selector;
+    app = new Video(client);
+  });
+
+  function verifyVideo(type, done) {
+    var title = 'VID_0001';
+    var filename = title + '.' + type;
+
+    client.fileManager.add({
+      type: 'videos',
+      filePath: 'test_media/Movies/' + filename
+    });
+    app.launch();
+    app.tapThumbnail(0);
+    // To avoid timing issues with the controls
+    // being auto-hidden when the video is first
+    // played, wait for controls to be hidden, then
+    // show them, by tapping on the video player.
+    setTimeout(function() {
+      app.showControls();
+      // Verify the video has begun playing and the
+      // title has been read properly.
+      var elapsedTime = app.elapsedTime();
+      assert(elapsedTime > '00:00');
```

Comment:
This assertion will pass even if the player is displaying 'undefined' or "NaN" instead of an actual time.
I think you want something like: `parseInt(elapsedTime.split(':').pop()) > 0`